core/src/SYCL/Kokkos_SYCL_KernelLaunch.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef KOKKOS_SYCL_KERNELLAUNCH_HPP_
+#define KOKKOS_SYCL_KERNELLAUNCH_HPP_
+
+#include <SYCL/Kokkos_SYCL_Error.hpp>
+#include <CL/sycl.hpp>
+
+template<class T>
+class kokkos_sycl_functor;
+
+#define CONCAT_IMPL( x, y ) x##y
+#define MACRO_CONCAT( x, y ) CONCAT_IMPL( x, y )
+#define CGH_PARALLEL_FOR cgh.parallel_for<class MACRO_CONCAT( kokkos_sycl_functor, __COUNTER__ )>
```

Comment:
These probably need to be prefixed with `KOKKOS_IMPL`

Reply:
Will do.